<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterWidget</class>
 <widget class="QDockWidget" name="FilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>738</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetClosable</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Filters</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="1" column="2">
     <widget class="QPushButton" name="delete_all">
      <property name="toolTip">
       <string>Remove all filters</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/Icons/Remove.png</normaloff>:/Icons/Remove.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="reduce">
      <property name="toolTip">
       <string>Reduce data set to filtered rows</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/Icons/ReduceData.png</normaloff>:/Icons/ReduceData.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="8">
     <widget class="QListWidget" name="filters">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="load_set">
      <property name="toolTip">
       <string>Load filter set</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/Icons/Open.png</normaloff>:/Icons/Open.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QPushButton" name="remove_set">
      <property name="toolTip">
       <string>Delete stored filter set</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/Icons/Remove.png</normaloff>:/Icons/Remove.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QPushButton" name="store_set">
      <property name="toolTip">
       <string>Store filter set</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/Icons/Save.png</normaloff>:/Icons/Save.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="4" rowspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="5" colspan="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Filter sets</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>103</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="enabled">
      <property name="toolTip">
       <string>Enable/disable filters</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Resources.qrc">
        <normaloff>:/Icons/Enabled.png</normaloff>:/Icons/Enabled.png</iconset>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Actions</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
